tests/containers/buildah_docker.pm

```diff
@@ -38,21 +41,22 @@ sub run {
         my $images_to_test = check_var('CONTAINERS_UNTESTED_IMAGES', '1') ? $untested_images : $released_images;
         for my $iname (@{$images_to_test}) {
             record_info "IMAGE", "Testing image: $iname";
-            test_container_image(image => $iname, runtime => 'buildah');
+            test_container_image(image => $iname, runtime => $buildah);
             if (check_os_release('suse', 'PRETTY_NAME')) {
                 # Use container which it is created in test_container_image
                 # Buildah default name is conducted by <image-name>-working-container
                 my ($prefix_img_name) = $iname =~ /([^\/:]+)(:.+)?$/;
-                test_opensuse_based_image(image => "${prefix_img_name}-working-container", runtime => 'buildah', version => $version);
+                test_opensuse_based_image(image => "${prefix_img_name}-working-container", runtime => $buildah, version => $version);
                 # Due to the steps from the test_opensuse_based_image previously,
                 # the image has been committed as refreshed
-                build_and_run_image(runtime => 'docker', buildah => 1, base => 'refreshed');
+
+                build_and_run_image(runtime => $docker, builder => $buildah, base => $iname);
             }
         }
     }
     scc_restore_docker_image_credentials();
-    clean_container_host(runtime => 'docker');
-    clean_container_host(runtime => 'buildah');
+    $docker->cleanup_system_host(0);
```

Comment:
0??

Reply:
do not validate cleanup. see above

Reply:
ah ok. It would be much easier to understand using %args: 
`$docker->cleanup_system_host(assert => 0);`

Reply:
let keep this open for reminder and if everything else approved that will be a small commit.